fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/api/event/client/player/ClientPickBlockCallback.java

```diff
@@ -20,14 +20,16 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.hit.HitResult;
 
+import net.fabricmc.fabric.api.client.interaction.v1.event.player.ClientPickItemEvents;
 import net.fabricmc.fabric.api.event.Event;
 import net.fabricmc.fabric.api.event.EventFactory;
 
 /**
  * This event handler has been deprecated due to not hooking nicely
- * into the game. Please use the alternatives.
+ * into the game and is no longer implemented. Please use the alternatives provided in {@link ClientPickItemEvents}.
```

Comment:
If this is no longer implemented, it needs to be removed, full stop.

Silent failing bugs are *impossible* to track down, please do not introduce one into fabric api.